pkgs/applications/networking/browsers/vivaldi/ffmpeg-codecs.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchurl, fetchpatch
+, dbus_glib, gtk2, gtk3, libexif, libpulseaudio, libXScrnSaver, ninja, nss
+, pciutils, pkgconfig, python2, xdg_utils
+}:
+
+stdenv.mkDerivation rec {
+  name = "${product}-${version}";
+  product = "vivaldi-ffmpeg-codecs";
+  version = "59.0.3071.104";
```

Comment:
The Chromium version has to match the one Vivaldi is using, so it should be updated in sync with the `vivaldi` package, not with the `chromium` one.